### DISCUSSION TITLE
personal/pneumothorax

### DISCUSSION THREAD

**Discussion by joe-antognini** [joe-antognini/joe-antognini.github.io#24] [Comments]
Written on 02/11/2020 07:54:26 

URL: https://joe-antognini.github.io/personal/pneumothorax

**Comment by joe-antognini** [comment]
Comment written by **Chris Robertson** on **12/17/2020 10:42:11**

<p>Hi. Joe. I happened upon your blog when I was searching for photos of collapsed lungs - I am in the UK and writing my history for my children and grandchildren and the incident of my collapsed lung was quite a standout episode. My story is very similar to yours except I walked down one side of a valley and up the other to go to Quaker Meeting. I thought I had indigestion. It was mid afternoon, when we were having tea with a friend who is a nurse, when she pointed out i was breathing 'funny'. We called our doctor and he agree to meet me at the surgery (it was a sunday) so I drove to meet him. There was no xray equipment there so he asked me to drive to the local hospital, some 15 miles away. He phoned ahead so I was taken in to xray straight away and then all hell broke loose. They had no specialists on call so I was ambulanced (blues and twos blazing) to the next hospital some fifty miles away!  When i got there they said my lung was down to 5% and did I want to wait while a local painkiller worked - they recommended that I didn't so they went ahead an needled me. They were no using suction in those days, just let nature take it's course, but ten days later, when they took the needle out the lung went straight back down - your experience explains that a bit. Eight days later the lung stayed up and I was released to home recovery. Thanks for sharing your story and thanks for the photos - they<br> will be great to help explain to my children (they were all under five at the time). Chris R</p>

**Comment by joe-antognini** [comment]
Comment written by **antognini** on **12/17/2020 19:43:44**

<p>Wow, thanks for sharing your story!  I am impressed that your lung eventually stayed up with no pleurodesis even if it took many days.  Did you have no further collapses after that?</p>

**Comment by joe-antognini** [comment]
Comment written by **Chris Robertson** on **12/18/2020 10:06:51**

<p>No, it has not been a problem over the thirty years since it happened. I occasionally get a twinge from that lung, just to remind me! I had smoked for years before the collapse and I am ashamed to say that I continued smoking for several years after. I was told that the smoking, stress and being tall and thin were all contributing factors. I think the phycological effects were greater than the medical - it was the first time I had been in hospital and the total lack of control over my life was difficult. Then coming out as an invalid for several weeks, being so helpless in front of my children was a struggle. Even now, with the covid pandemic, I consider myself at greater risk because of my 'history'. But I am alive and well and enjoying life so what more can you want? By the way, this was the sight that greeted me when I got home.  Chris R <a href="https://uploads.disquscdn.com/images/a447d48c51dd6da6aba34ab5bdef47f5d8fb5bc581f0fd2420b73a7193a8461a.jpg" rel="nofollow noopener" title="https://uploads.disquscdn.com/images/a447d48c51dd6da6aba34ab5bdef47f5d8fb5bc581f0fd2420b73a7193a8461a.jpg">https://uploads.disquscdn.c...</a></p>

**Comment by joe-antognini** [comment]
Comment written by **Kelly Davis** on **02/27/2021 10:20:33**

<p>..... I'm trying to find my words and type them out through tears.</p><p>As most nights after being home from the hospital, I'm on the internet trying to understand what happened to me. What's happening to me now. Everything I came across was about procedure, recovery and so on. I needed to know more specifically about how I was feeling mentally and what to expect.  At my first follow-up appointment with the pulmonary doctor he informed me that I had PTSD. It happened so suddenly. I was driving to work when it happened. It'd gotten a few calls from the office on my way in and they noticed I was having a hard time speaking. After explaining what was happening with the pain in my chest, shoulder and back and that my fingers were tingling a bit, they thought I was having a heart attack. An ambulance was called the minute they saw my car pulling on the lot. A few minutes later I was trying to convince the paramedics that I might be having a panic attack for some strange reason and they should go help people that really needed it. I would be ok.</p><p>Imagine my shock when the ER doctor tells me I have a spontaneous pneumothorax. A what? Did that mean I wasn't going to work?</p><p>I spent 5 days in the hospital and was released quickly because a snow storm was coming and they wanted me to get home. No one went over my discharge papers with me. I had no idea what to expect. Everything, the entire thing, happened so fast. So I've been searching the web for understanding. I needed a message board, a chat group, anything other than the text book definitions. Apparently, I wasn't typing in the right words or phrases. After my boyfriend checked my scars tonight we were trying to understand what was what. Why is this still swollen, what was that incision for, is this a stitch or a scar.... I decided to search for chest tube scars.</p><p>I saw your pictures and clicked on the article. I instantly felt like I wasn't crazy or alone. Your story and explanations are so spot on! You covered every concern I had. The sneezing, coughing, sleeping! The sleeping! My guy doesn't understand what I'm going through and why I'm more comfortable sleeping in a more upright position. He likes to think he can advise me, but he has no idea what he's talking about. He means well, but ugh.</p><p>Thank you so much for sharing your story. You've helped me more than you know. By the way, my experience happened February 9, 2021.</p>

**Comment by joe-antognini** [comment]
Comment written by **Edlyn Akino** on **02/27/2021 11:32:08**

<p> <a href="https://t.co/dTbE6Hztpi?lhhGvDtz" rel="nofollow noopener" title="https://t.co/dTbE6Hztpi?lhhGvDtz">https://t.co/dTbE6Hztpi?lhh...</a></p>

**Comment by joe-antognini** [comment]
Comment written by **antognini** on **03/05/2021 06:53:38**

<p>I'm sorry you went through all that!  But I'm glad that you found this post helpful --- when it happened to me I found a post that someone else had written that helped me me a lot and I decided to pay it forward.</p><p>I don't know if you've already found it, but there's a group on Facebook for people who have had a spontaneous pneumothorax (appropriately just called "Spontaneous Pneumothorax") and I've found it really helpful.</p><p>It takes a while to heal, but it all eventually does heal and you'll be back to very nearly 100%, so just know that things get better!</p>

**Comment by joe-antognini** [comment]
Comment written by **Adekvatan** on **03/07/2021 00:11:18**

<p>Trasnlate from Serbian languiage.</p><p>Ja sam takodje imao spontani pneumotoraks i kao i ti, opisaću moj problem u malo kraćem obliku i više ću da idem na poente. Tokom dana pre PneumoToraksa, u poslednjih mesec dana nisam trčao, plivao, gnjurio...Jedan dan sam pomagao oko nošenja nekih ormana, ali ništa nakon toga nisam osetio kao promenu, ali ajde da napomenem čisto reda radi. Zatim, dva dana pred dešavanja, sa ženom sam se šetao planinom i bio je vetar, spuštali smo se dole i gore i nekako sam se umarao, disao sam brzo i pritom je duvao vetar. Nisam obraćao pažnju jer smo hteli što pre u zaklon od vetra. Došao sam kući, ništa spektakularno nije bilo, proveo sam dosta vremena kući, a onda veoma kasno uveče legao da spavam. Sutradan sam se probudio sa osrednjim bolom u ledjima, bol u ledjima je bio na sredini, otprilike preko puta želudca i dijafragme. Probao sam da se razgibam, istegnem, ali bol je polako posle dva sata počeo da jača. Pozvao sam drugara da me izmasira profesionalno i dosta snažnije, što je on uradio. Nakon masaže ja sam apsolutno izgubio bol u ledjima, ali odjednom je krenulo da me boli umesto u ledjima, direktno u centar grudnog koša, negde kod dijafragme iznad želudca. Ja sam bio ubedjen da mi je želudac podigao dijafragmu. Uzeo sam šumeće tablete Magnezion i lek za želudac(sam na svoju ruku), očekivajući da će mi biti bolje, medjutim sve je bilo gore. Osećao sam neku kuglu ili lopticu na tom mestu, a kada sam hteo da udahnem, da loptica bi me bolela i morao bi da prestanem da udahnem vazduh, jer je bol daleko jači. Tako sam ja trpeo te bolove još dva sata dokle lek treba da radi 100%, posle toga nisam mogao ni da se presvučen, da skinem trenutku, obujem patike i to sve posle samo 6, 7 sati nakon budjenja ujutro. Sada je vreme Kovid, pa sam hitno kolima prebačen u urgentni centar u Srbiji, gde sam prvo čekao 2h da dodjem na red. Kada sam došao na red prvo su mi vadili krv i uradili sve potrebne testove SRCA, jer su prvo hteli da vide da li je srce u pitanju. Kada su ustanovili da nije srce, otišao sam na snimanje pluća i tamo su mi ustanovili da imam na vrhu pluća dva mehura i da mi je pukla plućna maramica, da imam tečnosti u plućima i da mi je desno plućno krilo nestalo. Odmah su mi spremili krevet, moja žena je krenula da paniči, brzo su to rešili i odveli me hitno doktoru. Dohvatio me je neki doktor i rekao, Miloše, sredićemo ovo, samo ne znamo koliko će vremena biti potrebno. Usput, ja imam 30 godina, zdrav sam veoma, bio sam sportista i trenutno od loših aktivnosti samo pušim jednu paklu cigareta dnevno. I tako, doktoru sam opisao prethodne dane i pošto kod njega već bolovi su mi se drastično povećali, nisam mogao da udahnem ni 30% vazduha, a da ne jecam od bolova. Doktor mi je kroz grudni koš(desnu sisu) sa prednje strane pod oštrim uglom ubacio neku malo deblju iglu, jeste bolelo, ali se veoma lako može izdržati i kroz tu iglu mi je dodatno ubrizgavao kiseionik. Ja sam tada bukvalno mogao da uvučem da kažemo 80% vazduha i bilo mi je mnogo bolje kada mi je to ubacio, nadisao sam se, napunio pluća, živnuo malo i odmah su me prebacili u sobu za ležanje da ospavam, pa sutra kreću akcije. Ne sećam se kako sam spavao tu prvu noć, ali verovatno ok, čim se ne sećam. Prvi dan posle prespavane noći kod lekara, vode me prvo na snimak, a onda kod istog doktora koji me je primio. Dr.Savić je taj specijalista koji mi je to uradio i rekao mi je ovako, Miloše sada ćemo pokušati najlakšom i najefikasnijom metodom, da ti vratimo plućno krilo i oporavimo disanje, tako što ćemo da ti probijemo ispod pzuha, na nivou desne sise na grudnom košu, dakle tu ćemo ti izbušiti i postaviti DREN, kroz taj dren ćemo pokušati da ispumpamo tečnost iz pluća, a to će stimulisati automatsko vraćanje plućnog krila. Doktor mi je dao lokalnu anesteziju, ali je mnogo bolelo, bukvalno sam se drao 3, 4 sekundi od bolova UUUUUUUUU, and then AAAAAAAAAAAAAAAA....Kada su mi ubacili dren, povezali su me za taj automat koji se priključi na straju i povremeno mi vadi vodu iz pluća. Ukupno za 5 dana mog lečenja izvukao mi je 500ml vode iz pluća. Ja nisam mogao u sedećem položaju da spavam više od 1-2 sata, meni je to bio pravi pakao. Ne možeš da se okreneš na bok levo, a ni desno, već samo u polu sedećem položaju na ledjima kako bi što više tečnosti izašlo iz pluća. Svaki dan su me skidali sa drena da bi me slikali i gledali da li napreduje moje plučno krilo, i da li se dešava ekspanzija. Posle prvog dana već se videlo minimalni napredak mog plućnog krila, drenaža je trajala oko 4-5 dana, ne sećam se baš tačno. Pošto sam ja strastveni pušač, popušio sam prvi dan TRI cigare, a kada su me spojili na dreniranje doktor mi je rekao nikako da ne popušim ni jednu, pa sam ispoštovao njegove zahteve. Kada se drenaža završila i pluća pružila ekspanziju, isključili su me sa drenaže ujutro u 7 sati. Ostao sam ceo dan i pratili su me da li imam tegobe, otežano disanje, bolove, sestre su mi stalno nudile Diklofen inekciju u gizicu za bolove, ja sam to svaki dan odbijao. Kada sam proveo ceo dan bez drena normalno, legao sam da spavam i čak sam spavao 3 sata, što je bilo čudo za taj položaj za mene. Naglasio bih da me je stalno bolelo rame na toj strani na kojoj mi je plućno krilo atrofiralo, jak bol u plećki i ramenu, ali sam izdržavao.  Kada sam se ujutro probudio, otišao sam na snimanje i doktor mi je rekao, ako bude dobar snimak na "LIVE rendgenu", to je kada me gledaju dok duboko dišem, dok gašnjem i dok držim udahnuta pluća. Taj snimak traje oko 20-30 sekundi i to prati najmanje TRI doktora koji se konsultuju. Videli su da mi je plućno krilo dobilo skoro potpunu ekspanziju od 95% i rekli su da će se to vratiti prirodno na 100% verovatno, jer je ovo spontani i kažu da sam jedan od rekordera po brzini oporavka ove faze lečenja. Sledeća faza bila je vadjenja drena iz dela ispod pazuha. To je jedan od gorih bolova, liči na neko veliko peckanje, prvo vade konce, a onda vade dren i veoma, veoma boli. Mazali su nekom masti da lakše kliza, ali je bol neverovatno bio jak, izdržiš jer moraš. Nakon toga, stavili su mi flastere na ranu i rekli mi da 30 minuta ležim bočno na strani na kojoj su mi stavljali dren, na strani na kojoj mi je otišlo plućno krilo(atrofiralo). Ležao sam pola sata, a onda su došli po mene da opet snime. Snimak je bio u redu, i taj dan su mi rekli da mogu na kućno lečenje. Otišao sam kući i još uvek imam strah od disanja dubokog da mi se nešto ne desi, još uvek imam kao malu lopticu od straha, ima se još uvek kao neka knedla u želudcu u sredini grudi, ali to je normalno su mi rekli. Pušio sam oko 10 cigara dnevno, doktor mi je rekao da dim od cigare utiče loše jer slabi te zidove, ali generalno ne utiče mnogo. Kaže da definitivno moram da smanjim cigare, ali da to nije razlog ovoga, već da je to bilo spontana reakcija nekoliko loših faktora u datoj sekundi u datom trenutku. Rekli su mi da ne pravim nagle korake, da ne dižem ništa teže od 3kg, da ne trčim i ne zamaram se, i da se ne prejedavam, da jedem dnevno više puta po malo, ali nikako da prenatrpavam želudac u tim prvim nedeljama. Ja sam se pridržavao toga, jeo sam kuvanu hranu, jeo sam četiri, pet puta dnevno po malo, bio sam razne vitamine i pušio sam 10 cigara dnevno. Posle mesec dana, vratio sam se na jednu paklu cigara dnevno. Ja sma strastveni pušač i rekao sam sebi, ako mi se ovo drugi put dogodi, 100% prestajem da pušim. Prošlo je već skoro godinu dana, sve je u redu. Na snimku sam imao potpuno atrofiralo desno plućno krilo i na snimku sam imao na vrhu pluća gore dva prstena, doktor mi je rekao da su to dva vazdušna mehura koja su ostala tu. Takodje, doktor mi je rekao, da u trenutku neke stresne situacije ili tokom teških snova, može se desiti toliko jaka reakcija da dolazi do tih teških okolnosti do kojih dodje u toku spavanja do havarije plućnog krila zbog prevelikog stresa i lučenja raznih ljudskih resursa odbrane. Prilikom tog lučenja, uzbudjenja, straha i svog adrenalina šta se proživljava u snu(nebitno dal se sećaš sna ili se ne sećaš), dolazi do pucanja toga, ide vazduh gore i udara u zid, i svaka pluća obično imaju i po neku cistu punu vode, i sve to pukne i desi se problem. Dakle, spontani Pneumotoraks se može desiti i kada su slaba genetski pluća, ali i kada se desi niz drugih faktora. Recimo, mom drugaru se desilo, kada je dizao preveliki teret, pa ga je držao i dodatno udahnuo vazduh i preseklo ga u plućima i tada mu je puklo. I on se oporavio nakon dve nedelje. Eto, hteo sam ovaj lakši oblik da vam prenesem. Ko god ovo ima, znamo svi mi koje muke trebaju da se predju, ali ti autoru ovog teksta, ti si baš prošao kroz ozbiljne muke. Ti si ovo na teži način preživeo, SAMO ZATO ŠTO SE NISI JAVIO NA VREME. Meni je doktor rekao, svaki bol koji traje duže od 2 sata intenzivno, odmah kod lekara hitno! Veliki pozdrav. Ovo je snimak preko prozora, izvinite što se loše vidi. Gore vidite te prstenove, to su vazdušni mehurovi, dva komada. Desno plućno krilo atrofiralo.... <a href="https://uploads.disquscdn.com/images/1ca619d20f71fbb07014a2784880265add5b6db6aedd83754583d6ebf47c994f.jpg" rel="nofollow noopener" title="https://uploads.disquscdn.com/images/1ca619d20f71fbb07014a2784880265add5b6db6aedd83754583d6ebf47c994f.jpg">https://uploads.disquscdn.c...</a></p>

**Comment by joe-antognini** [comment]
Comment written by **Steve Moore** on **03/07/2021 21:06:11**

<p>Yesterday I got out of the hospital after a 3-day stay due to a primary spontaneous pneumothorax (PSP).  I just finished reading your post about your experience and felt compelled to send you thanks and tell you my story, though it’s not quite as bad as yours.  Fortunately I didn’t have a pleurodesis, my lung recovered after the chest tube and vacuum.  My thoracic doc told me I had a 50% chance of having another PSP, which as you know was pretty uncomfortable, so I’m not too excited about that.</p><p>I was convinced I had covid, given the symptoms I was experiencing, shortness of breath, trouble breathing, chest pain, etc. and my plan was to ride it out like everybody else.  My wife thought it might be something else, possibly a cardiac event.  I’ve never had one, but I do see a cardiologist, and a friend of ours just died of a heart attack last week at age 47, I’m 50.  So she urged me to go the ER.  After 20+ years of marriage I knew better than to argue, so I rolled my eyes and let her drive me to the ER and drop me off, she couldn’t even wait with me.  I forced myself not to turn around, knowing she was probably crying, and thinking this might be the last time she ever sees me.</p><p>My story is pretty routine from this point on.  The ER doc told me I had a collapsed lung and I think he had to repeat it at least 3 times.  I’m not the picture of health, but I’m not overweight, I don’t smoke, and eat pretty healthy, although I could exercise more than I do.  They put in the chest tube, which I thought would be quite painful, but it wasn’t too bad.  I wish they had shaved my chest.  I think they forgot because I heard someone ask for a razor before they got started.  I asked the ER doc if he’d done this before and his reply was that it was his first one, “but I just watched a YouTube video, some I’m good to go!”.  Turns out he does a few every month.</p><p>It was strange seeing a tube sticking out of my chest and hearing the air whistling through the tube as I inhaled/exhaled before the pump was attached.  A chest x-ray a couple hours later showed my lung had reflated and was holding shape, unlike yours.  I stayed on the pump for about 30 hours, and my lung kept shape after the vacuum came off.  Then the tube came out about 24 hours later, (no humming for me, just a deep breath and exhale like you’re blowing out candles).</p><p>Knowing the odds of recurrence and the pleurodesis that would likely come after makes me pretty nervous.  After reading your story I feel a little better about it, although certainly hoping I’m in the favorable 50%.  I gave out a bit of a scream when I read that I wouldn’t be able to skydive though.  I have twin boys that are in college and we’re planning to skydive together once they graduate.  I’ve never done it before but have wanted to for years.  Apparently I can’t scuba dive (as you also mentioned), go down in a submarine, or do any alpine hiking in the near future either.</p><p>Anyway, thanks for sharing your story, very descriptive, informative, and educational.  I’ll refer back to it if I end up back in the ER with a second PSP.</p>

**Comment by joe-antognini** [comment]
Comment written by **Nicolas Zamorano** on **08/10/2021 09:29:53**

<p>Thanks for this, i really wanted someone who has gone through mostly what I went through to tell me that this takes time…</p><p>I’m a month in since the incident, and 2 weeks since surgery and I’m having a lot of trouble just sleeping.<br>I just can’t figure a way to sleep that will cause me the least amount of discomfort, and I’m scared shitless everytime i hear or feel anything on my left lung.</p><p>Anyway, thanks a lot again for taking the time to write bout it, as I’m getting tired of reading medical procedures with no real human touch of experience in them. Cheers.</p>

**Comment by joe-antognini** [comment]
Comment written by **antognini** on **08/11/2021 01:32:07**

<p>Glad this helped!  It's a sucky experience, there's no getting around that.  But it slowly gets better.  I was sleeping more or less upright for a while.</p>

**Comment by joe-antognini** [comment]
Comment written by **Luis Rizo** on **09/13/2021 20:57:17**

<p>Thank you for sharing. My wife has had multiple pneumothorax and is going to have surgery in a few weeks. We've read quite a lot but this post was very describing and informative.</p><p>If may ask, now that there's been some time. How are you today? Do you still feel pain? Can you fully excercice? Do you still feel any difference?</p>

**Comment by joe-antognini** [comment]
Comment written by **antognini** on **09/13/2021 21:26:14**

<p>The pain is 99.99% gone.  The main thing is that there is still some remaining numbness on my left chest.  (The surgery will cut through some nerves between the ribs and the nerves take a very long time to grow back and never fully recover.)</p><p>I am back to weightlifting, although I definitely go much lighter than I used to.  I haven't felt any issues while working out, though that could be be because I try not to push myself too much.  I think I get out of breath easier when I run, so I may have lost some lung capacity.  But I also may have just gotten out of shape while I was recovering.</p>

**Comment by joe-antognini** [comment]
Comment written by **Luis Rizo** on **09/14/2021 06:41:36**

<p>Thank you for your answers and again, thank you for this awesome post. It truly is helpful. You've helped more people than you can imagine</p>

**Comment by joe-antognini** [comment]
Comment written by **Cara Mrenak** on **09/24/2021 04:29:53**

<p>Thank you so much for sharing this. I can’t express how much better reading your experience has made me feel about everything. I’m a 27 year old mom who is absolutely terrified of everything right now. Just every day normal things, I’m so scared of this happening again that it has really put a strain on my life and my daughter’s. I too had it happen on my left side only it was a partial collapse (20% the first time and over 30% the second time almost exactly one month later). This just happened to me for the first time august 5th and again I was back in the hospital august 31st The first time it happened they did the “pigtail” which is a smaller chest tube they put right below my collarbone. The second time they pushed for surgery (I was absolutely petrified and had a panic attack when they told me I absolutely needed the pigtail the first time around, I don’t do well with these type of things and it was my first major surgery so just imagine how I reacted when they told me I needed this surgery). I am a very anxious person but also it doesn’t help that I’m 100lbs and I was terrified of any huge tube going into my tiny chest.  They did the VATS surgery but I only had 2 incisions. One under my armpit where they put the camera and the other between my ribs where they put the chest tube. They didn’t put a drain in, the box hooked to my chest tube drained the fluid and they used the talc to scar mine in place. Anyway, your story has helped me understand so much more than any doctor or nurse has explained to me. When I went to my post op appointment to get my staple and stitch removed and get my final x-rays from that doctor he assured me everything looked good but he also pointed out the blebs on my right side. He told me there was nothing I can do to heal them. He told me they could never get better but they could get worse or they could just stay there and not bother me ever. Since all this has happened I fell into a deep depression because my whole life turned upside down and it’s completely falling apart. Since my doctor told me about the blebs on my right side I’ve been absolutely terrified and I have constant anxiety and I’m just overall miserable. Hopefully now that I’ve heard your story when I get scared I can remind myself the things I’ve learned from this and comfort myself in knowing that so thank you again. You seriously have no idea how much this has helped me.</p>

**Comment by joe-antognini** [comment]
Comment written by **antognini** on **10/04/2021 15:55:34**

<p>I'm glad that this helped in a small way!  It's definitely not an easy thing to go through, but you will get better over time and life will start to return to normal.  It was definitely my experience that one of the harder parts was just not knowing what was going on.  It wasn't until two days after my surgery that one of the doctors sat down with me and explained what was happening.</p>

**Comment by joe-antognini** [comment]
Comment written by **Lauren** on **12/23/2021 04:38:26**

<p>Thank you for sharing your story. I’m currently considering my option of having pleurodesis done., I’ve had at this point 3 pneumothoraces in my left lung- a unfortunate long tail result of having gotten Covid in the early part of Spring. This gives me a good idea Of what to possibly expect with the surgery.</p>

**Comment by joe-antognini** [comment]
Comment written by **Karlie Collins** on **02/19/2022 03:12:42**

<p>You are the reason I went to the ER yesterday morning. I am also 18 weeks pregnant so not only did your blog push me to go because all my symptoms lined up with yours but I knew i had to go straight to the ER rather than trying to meet with my pcp due to how sensitive the issue is.</p><p>Thank you so much for writing this. The professionals are surprised since spontaneous pneumothorax is uncommon in pregnancy so I am glad I was confident after reading your blog.</p><p>I thought you would be encouraged to know that it potentially saved two lives (especially my baby’s if I had stopped getting adequate oxygen but thankfully it stayed close to 100 the entirety of this process)</p><p>Thank you very much <br>K</p>

**Comment by joe-antognini** [comment]
Comment written by **antognini** on **02/21/2022 06:26:06**

<p>Wow, that is so scary! I'm so glad that you got the ER quickly!  I hope you aren't in too much pain and that the baby is doing well.</p>

**Comment by joe-antognini** [comment]
Comment written by **Sofiennnn** on **03/27/2022 18:54:10**

<p>I underwent the same surgery that you did, I have two weeks of discharge from the hospital, but every night I feel an infection in my lungs. And I feel tired when I walk, is this normal? Thank you for sharing your experience</p>

**Comment by joe-antognini** [comment]
Comment written by **antognini** on **03/27/2022 19:25:13**

<p>The healing process takes much longer than two weeks.  I was feeling pain and tiredness for several months.  So what you're feeling sounds normal.  But if you start to have shortness of breath I'd go to the doctor right away to make sure that the lung didn't collapse again.</p>

**Comment by joe-antognini** [comment]
Comment written by **Sofiennnn** on **03/27/2022 20:50:15**

<p>I did an x-ray and the doctor told me everything was fine. He could not explain the reason for this shortness of breath</p>

**Comment by joe-antognini** [comment]
Comment written by **Sofiennnn** on **03/27/2022 20:51:59**

<p>When I rest I feel fine but when I move I get tired and breathe slowly</p>

**Comment by joe-antognini** [comment]
Comment written by **Anthony g** on **09/17/2022 16:37:33**

<p>Have you done any air travel during the past 30 years since initial occurrence ?</p>

**Comment by nicoleishere723** [comment]
Thank you so much for this. I am currently dealing with a right lung collapse x2 (also here in SF), and am wondering what surgery would be like. 

Thank you for this insight. I hope you feel better as this is extremely painful.

**Reply by nicoleishere723** [reply]
For reference, I have done a chest tube and needle aspiration a couple of months ago. So this has been quite the journey.
